content/docs/command-reference/exp/run.md

```diff
@@ -137,19 +141,21 @@ CPU cores).
   name will generated by default, such as `exp-f80g4` (based on the experiment's
   hash).
 
+- `--temp` - run this experiment outside your workspace (in `.dvc/tmp/exps`).
+  Useful to continue working (e.g. in another terminal) while a long experiment
+  runs.
+
 - `--queue` - place this experiment at the end of a line for future execution,
-  but do not actually run it yet. Use `dvc exp run --run-all` to process the
+  but don't actually run it yet. Use `dvc exp run --run-all` to process the
   queue. For checkpoint experiments, this implies `--reset` unless a `--rev` is
   provided.
 
-- `--run-all` - run all queued experiments (see `--queue`). Use `-j` to execute
-  them [in parallel](#queueing-and-parallel-execution).
+- `--run-all` - run all queued experiments (see `--queue`) and outside your
+  workspace (in `.dvc/tmp/exps`). Use `-j` to execute them
+  [in parallel](#queueing-and-parallel-execution).
 
 - `-j <number>`, `--jobs <number>` - run this `number` of queued experiments in
-  parallel. Only applicable when used in conjunction with `--run-all`.
```

Comment:
What's the `-j` default here, BTW? Also 4 x CPU cores like in other commands?

Reply:
Default is 1. See https://github.com/iterative/dvc/pull/5785.

Reply:
> Default is 1.

Thanks! Noted in eab608f7.